packages/ListBox/src/components/Option/Option.styles.js

```diff
@@ -30,13 +31,13 @@ export const OptionStyled = styled.li`
 
     const isActived = props.isActive
       ? `
-        border: 3px solid Highlight;
+        border: 2px solid Highlight;
```

Comment:
I don't think `Highlight` is valid for `border-style`.

Reply:
I checked it in dev tool, `Highlight` is valid. I don't know why we use `Highlight` though. Can I confirm with @johnpeele and update it later?